rom/dev/tests/test_idevid_derivation.rs

```diff
@@ -41,6 +41,9 @@ fn test_generate_csr() {
 
     let output = String::from_utf8_lossy(&output);
     assert_output_contains(&output, "[idev] CSR uploaded");
+
+    // Remove the CSR file.
+    let _ = fs::remove_file("caliptra_idevid_csr.der");
```

Comment:
I suggest putting a deletion helper function over next to https://github.com/chipsalliance/caliptra-sw/blob/59c457b2e328ef89dfec05851e883fd035ab9e35/sw-emulator/lib/periph/src/soc_reg.rs#L724,L746 where these files are created, so the file paths, creation and deletion, is concentrated in one place.

Plus from that file it looks like there are two DER files that may be generated, though I'm not clear on why caliptra_ldevid_cert.der is not being created.

Reply:
Can we move this file into some junk directory that's in .gitignore, and out of the main source directories?

Reply:
+1, if soc_reg.rs were tweaked to initially generate these files in an ignored directory that would be great.

Reply:
Removed auto-generation of the CSR file in the emulator. Tests now have to explicitly download the CSR from the mailbox.